src/ga4gh/vrs/models.py

```diff
@@ -474,8 +523,21 @@ class Allele(_VariationBase):
         ..., description='An expression of the sequence state'
     )
 
+    def ga4gh_serialize_as_version(self, as_version):
+        """This method will return a serialized string following the conventions for
+        Allele serialization as defined in the VRS version specified by 'as_version`."""
+        location_digest = self.location.compute_digest(as_version=as_version)
+        sequence = get_pydantic_root(self.state.sequence)
```

Comment:
We may want to check that `state` is a `LiteralSequenceExpression` and raise exception if not. `LengthExpression` does not have a `sequence` field and not sure if we want `ReferenceLengthExpression` to be converted to `LSE` in 1.3

Reply:
LSE _or_ RLE is okay here; both have a `sequence` field, and for small variants the RLE `sequence` is populated by default. However, I agree that we want to do some processing here as a separate PR; ideally we would implement the RLE to LSE function (it should work [like this](https://github.com/ga4gh/vrs/issues/426#issuecomment-1828387956)) and just apply that in the case this is an RLE without a defined sequence.